webrtc/_compositions/go2go.toml

```diff
@@ -0,0 +1,64 @@
+[metadata]
+  name = "go2go {{ .Env.GitReference }}"
```

Comment:
Is that something they do? I was for now just giving it a static name

Reply:
Yes that came from the existing one. That said, the existing one is doing something I'm not - they're iterating over _different_ versions of each codebase and testing their interoperability against each other.

That's not a bad idea to do... but since I haven't done it yet that extra tagging is unnecessary complexity for now.